src/libmeasurement_kit/ooni/constants.hpp

```diff
@@ -56,6 +56,8 @@ const http::Headers COMMON_CLIENT_HEADERS = {
   }
 };
 
+const std::string MEEK_SERVER_RESPONSE = "I\u2019m just a happy little web server.\n";
```

Comment:
Is `\u2019` a unicode character? If so, how wide are characters in this string? Are they UTF-16?

(I would have expected the server to return UTF-8, so I'm asking &mdash; and I guess that in UTF-8 that apostrophe is composed by two 8 bit characters, no?)